### PR TITLE
[15.0][MIG] website_sale_attribute_filter_order > website_sale_product_attribute_filter_order: Migration to version 15.0

### PR DESCRIPTION
**Pull request by pilarvargas-tecnativa**
cc @Tecnativa TT37010

@chienandalu @CarlosRoca13 please review :)

### CODE REVIEW

**Review by chienandalu — COMMENTED** [review]
Are you sure this isnt't a core functionality already?

**Review by chienandalu — APPROVED** [review]
Good job :)

File: website_sale_product_attribute_filter_order/README.rst [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+#. Go to website (frontend)
+#. Drop down the 'Customize' menu
+#. Enable 'Product Attribute's Filters Marked First' option
+
+Usage
+=====
+
+#. Go to shop
+#. Active one or more attributes
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_order/README.rst]:
```suggestion
#. Set one or more attributes on
```

File: website_sale_product_attribute_filter_order/README.rst [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+=============
+
+#. Go to website (frontend)
+#. Drop down the 'Customize' menu
+#. Enable 'Product Attribute's Filters Marked First' option
+
+Usage
+=====
+
+#. Go to shop
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_order/README.rst]:
```suggestion
#. Go to the eCommerce
```

File: website_sale_product_attribute_filter_order/README.rst [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+#. Go to website (frontend)
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_order/README.rst]:
```suggestion
#. Go to the eCommerce (/shop)
```

File: website_sale_product_attribute_filter_order/README.rst [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+#. Drop down the 'Customize' menu
+#. Enable 'Product Attribute's Filters Marked First' option
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_order/README.rst]:
```suggestion
#. Enable the 'Product Attribute's Filters Marked First' option
```

File: website_sale_product_attribute_filter_order/README.rst [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+Usage
+=====
+
+#. Go to shop
+#. Active one or more attributes
+#. You can see that all enabled attributes are moved to the first position of the list
```
Comment (chienandalu) [inline website_sale_product_attribute_filter_order/README.rst]:
```suggestion
#. You can see that all enabled attribute values are bubbled up to the first position in the list.
```

**Comment by pilarvargas-tecnativa** [comment]
@chienandalu I've already made the changes ;)

**Review by chienandalu — APPROVED** [review]
Thanks!

**Review by CarlosRoca13 — APPROVED** [review]

**Pull request merged**